components/eamxx/src/share/io/scream_output_manager.cpp

```diff
@@ -282,6 +285,22 @@ add_global (const std::string& name, const ekat::any& global) {
 }
 
 /*===============================================================================================*/
+void OutputManager::init_timestep (const util::TimeStamp& start_of_step)
+{
+  // In case output is disabled, no point in doing anything else
+  if (not m_output_control.output_enabled()) {
+    return;
+  }
+
+  if (m_atm_logger) {
+    m_atm_logger->debug("[OutputManager::init_timestep] filename_prefix: " + m_filename_prefix + "\n");
+  }
+
+  for (auto s : m_output_streams) {
+    s->init_timestep(start_of_step);
+  }
```

Comment:
Yes, it does. We need to change the signature of `init_timestep` then, and pass also `end_of_step` (or, equivalently, `dt`), since the end-of-step timestamp is what is used to determine if output will happen.